### DISCUSSION TITLE
Website works on computer, but shows a '404 There isn't a GitHub Pages site here' error

### DISCUSSION THREAD

**Discussion by kaylajo838** [community/community#39778] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I set up my portfolio website and attached the live preview link to the href links for my projects. On my computer, everything works flawlessly. On anyone's phone, all but one page show the '404 There isn't a GitHub Pages site here' error. I have gone back and checked my  code, checked my deployment branches and folders, checked for typos, added /index.html to the end of the link, and I've also tried routing the href through my local system through files instead of the live link. Nothing has worked so far. I've scoured the internet for hours and can't find a fix that is working. Can anyone help?

**Comment by radar-blip** [comment]
Its working fine for me.
What page is it that is not working?

**Reply by kaylajo838** [reply]
I got it working! Turns out I accidentally had my actual project in the docs folder instead of root. I changed it and everything is now working as it should.

**Reply by radar-blip** [reply]
Awesome! 🎉
